src/common/log.js

```diff
@@ -2,6 +2,7 @@
 
 const { join } = require('path')
 const { assign } = Object
```

Comment:
Ah, yes, I didn't see how it was used, sorry -- in the rare case where you need to modify an object in place assign is good.